ui/00_message/ui/Photo_Upload.xml

```diff
@@ -22,19 +22,24 @@
   <message>
     <key>pu_msg_004</key>
     <original>写真送信中</original>
+    <translation>Sending Photo</translation>
   </message>
   <message>
     <key>pu_msg_005</key>
     <original>電源を切らないでください</original>
+    <translation>Please do not turn off the power</translation>
   </message>
   <message>
     <key>pu_msg_009</key>
     <original>現在サーバーが混み合っています
 しばらく経ってから再度お試しください</original>
+    <translation>The server is busy.
```

Comment:
Should probably have a period at the end.

Reply:
Granted I'm incredibly new, any reason to not include the "please wait and try again later" bit?

Reply:
If that is in the original text, then yes, it should be included. Given the amount of messages being contributed, I can't verify every translation for correctness other than checking over formatting.

Reply:
No problem. It was equal parts question to you as it was to Sole. Wasn't sure if there was a space restriction or anything prevent translation of the full message.

No, if the full translation is now included, I hadn't seen it. Assuming formatting is good, that was the only thing I wasn't sure on.

Reply:
There are restrictions depending on the text and location of the text. In general, DDON UI does not handle English text very well. There is no support for word wrapping whatsoever, only breaks on a character-to-character basis, and not all UI elements do word wrapping (some scale text down).

For this specific situation, there is no issue with the full message. You probably didn't see the second line because this comment was made on the first line. If you look at the entire file you can see it.

Reply:
No, you're right. I hadn't seen it until after I wrote that previous comment. I saw it later and understood what you meant. Thanks for clarifying.